docs/rules/prefer-ember-test-helpers.md

```diff
@@ -0,0 +1,66 @@
+# prefer-ember-test-helpers
+
+This rule ensures the correct Ember test helper is imported when using methods that have a native window counterpart.
+
+There are currently 3 Ember test helper methods that have a native window counterpart:
+
+* blur
+* find
+* focus
+
+If these methods are not properly imported from Ember's test-helpers suite, the native window method version is used instead, and any intended asynchronous functions won't work as intended, which causes tests to fail silently.
+
+See the origin and background of the rule proposal: [eslint-plugin-ember issue 676](https://github.com/ember-cli/eslint-plugin-ember/issues/676)
+
+## Examples
+
+Examples of **incorrect** code for this rule:
+
+```js
+test('foo', async (assert) => {
+  await blur();
```

Comment:
These functions should always have something passed to them right? We should use `blur('.some-element')` to make this a bit more realistic.

Reply:
Ah duh good point ;)